code/modules/clothing/masks/boxing.dm

```diff
@@ -18,7 +18,7 @@
 	desc = "Worn by robust fighters, flying high to defeat their foes!"
 	icon_state = "luchag"
 	item_state = "luchag"
-	flags = BLOCKHAIR
+	flags = BLOCKHAIR | MASKCOVERSMOUTH
 	flags_inv = HIDEFACE
 	w_class = 2

```

Comment:
Dorsi, but I have seen PLENTY of those full face masks without the mouth part. Besides, I don't see them on the sprite...sooo